lua/starfall/libs_sh/holograms.lua

```diff
@@ -383,15 +383,13 @@ else
 		if parent ~= nil then
 			parent = getent(parent)
 			if bone ~= nil then checkluatype(bone, TYPE_BOOL) end
-			if attachment ~= nil then
+			if attachment ~= nil or bone then
 				if isstring(attachment) then
 					attachment = bone and parent:LookupBone(attachment) or parent:LookupAttachment(attachment)
```

Comment:
Gotta change this to an if-else since LookupBone returns nil instead of -1

Reply:
Well yeah, I pointed this [here](https://github.com/thegrb93/StarfallEx/pull/1281#issuecomment-1069096296), just thought you didn't care about such case or it was impossible. Changed it to an `else-if` now.

Reply:
Wrong name of the last commit, actually fixes edge case where `bone` is used, but attachment is found instead and parents to a random bone